src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -20,99 +20,534 @@ PICS:
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: (!S.S.C40.Rsp) && PICS_SKIP_SAMPLE_APP
+      arguments:
+          values:
+              - name: "message"
+                value:
+                    "Please execute the add scene command with
+                    extensionfieldsets in accordance with AC1 on DUT and enter
+                    'y' if the command is successful"
+              - name: "expectedValue"
+                value: "y"
 
     - label:
           "TH sends a EnhancedViewScene command to DUT with the GroupID field
           set to G1 and the SceneID field set to 0x01."
-      PICS: S.S.C41.Rsp
-      verification: |
-          DUT sends a EnhancedViewSceneResponse command to TH with the Status field set to 0x00 (SUCCESS), the GroupID field set to G1, the SceneID field set to 0x01, the TransitionTime field set to 0x000a (1s) and a set of extension fields appropriate to AC1.
-      disabled: true
+      PICS: S.S.C41.Rsp && PICS_SDK_CI_ONLY
+      command: "EnhancedViewScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "GroupID"
+                value: 0x0001
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: 0x000a
+              - name: "ExtensionFieldSets"
+                value:
+                    [
+                        {
+                            ClusterID: 0x0006,
+                            AttributeValueList:
+                                [{ AttributeID: 0x0000, AttributeValue: 0x01 }],
+                        },
+                        {
+                            ClusterID: 0x0008,
+                            AttributeValueList:
+                                [
+                                    {
+                                        AttributeID: 0x0000,
+                                        AttributeValue: 0x64,
+                                    },
+                                    {
+                                        AttributeID: 0x0001,
+                                        AttributeValue: 0x00,
+                                    },
+                                ],
+                        },
+                    ]
 
     - label:
           "TH sends a ViewScene command to DUT with the GroupID field set to G1
           and the SceneID field set to 0x01."
-      PICS: "!S.S.C41.Rsp"
-      verification: |
-          DUT sends a ViewSceneResponse command to TH with the Status field set to 0x00 (SUCCESS), the GroupID field set to G1 and the SceneID field set to 0x01, the TransitionTime field set to 0x0001 (1s) and a set of extension fields appropriate to AC1.
-      disabled: true
+      PICS: (!S.S.C41.Rsp) && PICS_SDK_CI_ONLY
```

Comment:
Please file another bug on the test plan.  The test plan should be testing ViewScene even if EnhancedViewScene is supported.

It should also be testing AddScene (followed by EnhancedViewScene if that's supported) even if EnhancedAddScene is supported.  These things are not mutually exclusive and a device can support both.